### ISSUE TITLE
copy "." doesn't tolerate symlinks

### ISSUE THREAD

**Issue by errordeveloper** [erikh/box#167]
I've seen this:
```
[Boxfile] !!! Error: Rel: can't make . relative to ../../examples
```
...and it was in my project, so I've worked around it.
But my project has `vendor/github.com/docker/docker` and that resulted in:
```
[Boxfile] !!! Error: Rel: can't make . relative to ../../../contrib/init/sysvinit-debian/docker.default
```
I've tried using `ignore_list`, but it didn't seem to work, so I ended-up deleting those symlinks manually.

I'm looking into fixing this.

**Comment by erikh** [comment]
it's actually broken in docker; it uses a relative symlink which should be
a big no-no. I don't know what the best way to resolve this specific issue
is.

I just usually delete the symlink.

If you want to take a stab at a patch, thanks!

**Comment by errordeveloper** [comment]
> it's actually broken in docker; it uses a relative symlink which should be
a big no-no. I don't know what the best way to resolve this specific issue
is.

Yeah, symlinks in git give bad (well, the only thing that I'm aware of if is that they are broken on Windows). In my project I used symlinks cause I was lazy (well, I still am, I've just moved creation and deletion of those symlinks into a Makefile).

> If you want to take a stab at a patch, thanks!

Should we simply ignore and warn?

**Comment by erikh** [comment]
that seems good to me.

**Comment by erikh** [comment]
if you could also document the behavior for the copy verb, I would really
appreciate it.

**Comment by erikh** [comment]
Fixed by #168
